src/referral/serializers/referral_invite_serializer.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.serializers import ModelSerializer
+
+from referral.models import ReferralInvite
+
+
+class ReferralInviteSerializer(ModelSerializer):
+    class Meta:
+        model = ReferralInvite
+        fields = [
+            "id",
+            "inviter",
+            "recipient",
+            "recipient_email",
+            "invite_type",
+            "created_date",
+            "recipient_email",
+            "referral_first_name",
+            "referral_last_name",
+            "unified_document",
+        ]
+
+    def create(self, validated_data):
```

Comment:
@le-sun I ended up leaving this in place.

If I try to get rid of it, I get 

```
django.db.utils.IntegrityError: null value in column "expiration_date" of relation "referral_referralinvite" violates not-null constraint
DETAIL:  Failing row contains (42, 2022-11-10 01:49:46.822459+00, 2022-11-10 01:49:46.822483+00, f, null, , kobeattias@gmail.com, JOIN_RH, 8, null, null, Yaacov, Attias).
```

because the parent `Invite` class that is responsible for the `expiration_date` logic isn't being called. Not sure if there is anything to do here